test/Makefile

```diff
@@ -312,7 +323,10 @@ run-css: run-mongo-db
 		--name "$(DOCKER_CSS_CNAME)" \
 		-p 127.0.0.1:9443:9443 \
 		--network "$(DOCKER_TEST_NETWORK)" \
-		-e "HZN_EXCHANGE_URL=http://$(DOCKER_EXCH_CNAME):8080/v1" \
+		-e "HZN_EXCHANGE_URL=$(DOCKER_EXCH)" \
+		-e "CSS_URL=$(CSS_URL)" \
```

Comment:
I know we need the HZN_EXCHANGE_URL set, do we need the other 3 for the CSS container?

Reply:
You are correct, we do not need the other 3 for the CSS container. I just ran a test to confirm.